package.json

```diff
@@ -21,14 +21,14 @@
   "author": "Hendrik Muhs",
   "license": "MIT",
   "dependencies": {
-    "@actions/cache": "^1.0.4",
-    "@actions/core": "^1.2.6",
-    "@actions/exec": "^1.0.4",
-    "@actions/io": "^1.0.2"
+    "@actions/cache": "^1.0.7",
```

Comment:
https://github.com/actions/toolkit/blob/main/packages/cache/RELEASES.md#105

I'm having some trouble on the windows cpython builds